public/app.js

```diff
@@ -216,5 +217,36 @@
         });
 
         $httpProvider.interceptors.push('jwtInterceptor');
-    }
+
+        $translateProvider
+            .useSanitizeValueStrategy('sanitizeParameters')
+            .useStaticFilesLoader({
+                prefix: 'static/languages-<%= VERSION %>/',
+                suffix: '.json'
+            })
+            .determinePreferredLanguage(function () {
+                let lang = 'en'; // default language
+
+                if (navigator.language) {
+                    lang = navigator.language;
```

Comment:
From a little reading around this, we might want something different for IE users. `navigator.userLanguage` seems to be IE's option. 

We could use something like `navigator.userLanguage || navigator.language` to handle both. 

(FWIW, Google Analytics shows that 5% of logged in sessions are from IE - old IE, not Edge)

Reply:
Okay, I can make a more robust method. There is one built into the library (https://angular-translate.github.io/docs/#/guide/07_multi-language#multi-language_determining-preferred-language-automatically) but I wanted to extend it with the query string override (great for testing)